java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/persistence/Metastore.java

```diff
@@ -30,4 +30,9 @@
    */
   boolean store(String keyId, Instant created, V value);
 
+  /**
+   * Returns the region suffix for the metastore or "" if region suffix option is disabled
```

Comment:
description can be just `Returns the region suffix or "" if region suffix option is disabled`